runelite-api/src/main/java/net/runelite/api/WorldType.java

```diff
@@ -75,6 +76,13 @@
 		this.mask = mask;
 	}
 
+	public static EnumSet<WorldType> PVP_WORLD_TYPES = EnumSet.of(
```

Comment:
Maybe this should be private? or maybe each world could have flag if its PVP and you would build this enumset based on that

Reply:
Wasn't sure about exactly how to go about this--just knew I wanted to move it into WorldType since I'll actually need this for another PR. I'd be fine with making it private or making it an enum flag, though I slightly prefer just making it private.

Reply:
Alright I guess we can go with that

Reply:
Done